scripts/remote_customize_entrypoint.sh

```diff
@@ -41,18 +41,20 @@ setup_resolv() {
     # Copy the container's /etc/resolv.conf to the image root for customization
     local IMAGE_ROOT_RESOLV=$IMAGE_ROOT_DIR/etc/resolv.conf
     local TMP_RESOLV=$IMAGE_ROOT_DIR/tmp/resolv.conf
-    if [ -f "$IMAGE_ROOT_RESOLV" ]; then
+    if [ -f "$IMAGE_ROOT_RESOLV" -o -L "$IMAGE_ROOT_RESOLV" ]; then
         mv "$IMAGE_ROOT_RESOLV" "$TMP_RESOLV"
+    else
+        echo "Did not find an existing /etc/resolv.conf file"
     fi
-    cp /etc/resolv.conf "$IMAGE_ROOT_RESOLV"
+    cp --remove-destination /etc/resolv.conf "$IMAGE_ROOT_RESOLV"
```

Comment:
What benefit does adding the --remove-destination option confer? cp normally overwrites the destination file.

Reply:
The original problem popped up because cp won't overwrite a broken symlink. That caused all subsequent operations to fail since the k8s resolv.conf didn't get put in place.

This should not be needed, but is an extra backup to insure the k8s resolv.conf gets put in place even if there is a hiccup with moving the original version.